src/shared/constants.h

```diff
@@ -8,6 +8,8 @@
 /* Game Rules */
 // The max allowed speed of the ball, in metres per second
 const double BALL_MAX_SPEED_METERS_PER_SECOND = 6.5;
+// The max allowed height of the robots, in metres
+const double ROBOT_MAX_HEIGHT_METERS = 0.15;
```

Comment:
Just curious, why do we have this defined as a max rather then `ROBOT_HEIGHT_METERS`?

Reply:
it's the legal max according to SSL rules. It's not necessarily meant to be _our_ height.